client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -103,7 +103,7 @@ class BusinessDetails extends Component {
 				extensionInstallationOptions[ 'mailchimp-for-woocommerce' ],
 			install_mailpoet: extensionInstallationOptions.mailpoet,
 			install_jetpack: extensionInstallationOptions.jetpack,
-			install_google_ads:
+			install_google_listings_and_ads:
```

Comment:
If we're switching this to match the plugin name, maybe we can dynamically loop over the plugins to generate this track info.  What do you think?

Reply:
I'd agree with you. Although I can only speak for the desired tracks event for our plugin. There are other event names that don't match the plugin names, like `mailchimp`. I don't think it's desirable to change those at the moment.